AK/Assertions.cpp

```diff
@@ -10,10 +10,14 @@
 #include <AK/StringBuilder.h>
 #include <AK/StringView.h>
 
-#if (defined(AK_OS_LINUX) && !defined(AK_OS_ANDROID)) || defined(AK_LIBC_GLIBC) || defined(AK_OS_BSD_GENERIC) || defined(AK_OS_SOLARIS) || defined(AK_OS_HAIKU)
+#if __has_include(<execinfo.h>) && __ANDROID_API__ >= 33
 #    define EXECINFO_BACKTRACE
 #endif
```

Comment:
This does not look right.

`__ANDROID_API__` is undefined on other platforms, so this comparison will be treated as `0 >= 33` on Linux, for example. How about dropping the check from this `#if`?

```suggestion
#if __has_include(<execinfo.h>)
#    define EXECINFO_BACKTRACE
#endif
```